draft-patton-cfrg-vdaf.md

```diff
@@ -705,6 +705,159 @@ TODO
 * Output shares are secret shares of a vector of field elements, each
   corresponding to a counter for one of the candidate prefixes.
 
+## Dependencies
+
+[TODO Clarify which of these can be assumed and which need to be explicitly
+defined]
+
+### Descriptions of finite fields
+
+We assume a structure for describing IDPF output groups, named `ValueType`.
+For verifiable heavy hitters, we need an IDPF that at each level has two field
+elements as outputs, and so the value type would be pairs of field elements.
+The IDPF key generation and evaluation functions will allow a different `ValueType`
+parameter per bit in the IDPF domain, which allows us to use different types at
+the output layer, required for extractability. See [BBCGGI21, Section 4.3] for
+details. We assume a function `get_value_type(n: Unsigned) -> ValueType` that
+returns the type for bit index `n`. We use `ValueType` to refer to a description
+of a given group, and`Value` to refer to an actual element of the group.
+
+### Incremental Distributed Point Functions (IDPFs)
+
+An IDPF is defined over a domain of size `2**d`. The client can specify an
```

Comment:
I know we're supposed to be doing everything in Python, but let's just write this as
```suggestion
An IDPF is defined over a domain of size `2^d`. The client can specify an
```